### PR TITLE
Ccvi sub-scores component

### PR DESCRIPTION
**Pull request by chasulin**
[Mocks](https://www.figma.com/file/9c8BXfHbh8hKfLitPhW7Yn/CAN-Vulnerabilities?node-id=108%3A2328)

Adds table with specific vulnerabilities scores + tooltips: 
Screen Shot 2021-02-26 at 10 00 00 PM

To QA: 
`yarn storybook`
`vulnerabilities` folder under `shared components`

### CODE REVIEW

File: src/components/VulnerabilitiesBlock/VulnerabilitiesBlock.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import VulnerabilitiesBlock from './VulnerabilitiesBlock';
+
+export default {
+  title: 'Shared Components/Vulnerabilities/VulnerabilitiesBlock',
+  component: VulnerabilitiesBlock,
+};
+
+export const Block = () => {
```
Comment (chasulin) [inline src/components/VulnerabilitiesBlock/VulnerabilitiesBlock.stories.tsx:9]: Ignore this file for now. It's doing the same thing as `SubscoresBlock.stories.tsx`

File: src/components/VulnerabilitiesBlock/VulnerabilitiesBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import SubscoresBlock from './SubscoresBlock/SubscoresBlock';
+import { useCcviForFips } from 'common/hooks';
+
+const VulnerabilitiesBlock = () => {
```
Comment (chasulin) [inline src/components/VulnerabilitiesBlock/VulnerabilitiesBlock.tsx:5]: Ditto to this component for now

File: src/components/VulnerabilitiesBlock/SubscoresBlock/SubscoresBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import { orderedCcviThemes, CcviThemeInfo } from 'cms-content/tooltips';
+import { InfoTooltip } from 'components/InfoTooltip';
+import { RegionCcviItem } from 'common/data';
+import { getCcviLevel, getCcviLevelColor, getCcviLevelName } from 'common/ccvi';
+
+export interface ThemeWithScore extends CcviThemeInfo {
+  score: number;
+}
+
+const SubscoresBlock: React.FC<{ scores: RegionCcviItem }> = ({ scores }) => {
```
Comment (chasulin) [inline src/components/VulnerabilitiesBlock/SubscoresBlock/SubscoresBlock.tsx:12]: I've been using 'theme' and 'subscore' interchangeably, which might be confusing?

File: src/components/VulnerabilitiesBlock/SubscoresBlock/SubscoresBlock.style.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+export const LevelCell = styled.div<{ color: string }>`
+  ${CellStyles};
+  background-color: ${({ color }) => color};
+  color: white;
+  max-width: 80px;
+  padding: 12px 8px;
+  font-weight: 500;
+  font-size: 0.7rem;
+  text-transform: uppercase;
```
Comment (pnavarrc) [inline src/components/VulnerabilitiesBlock/SubscoresBlock/SubscoresBlock.style.tsx:31]: This is a bit weird, VoiceOver spells some levels intead of reading them. Can you add the level text in lowercase as `aria-label` for the component?

**Review by pnavarrc — APPROVED** [review]
Looks good!

File: src/components/VulnerabilitiesBlock/SubscoresBlock/SubscoresBlock.style.tsx (hunk 4 above)
Comment (pnavarrc) [reply src/components/VulnerabilitiesBlock/SubscoresBlock/SubscoresBlock.style.tsx:31]: It's probably because it's shown in uppercase

**Pull request merged**